spec/games_spec.rb

```diff
@@ -0,0 +1,55 @@
+require_relative 'spec_helper'
+require './lib/games'
+
+RSpec.describe Games do
+    before(:each) do
+        @game_objects = []
+
+        #iterate over the csv file to create an array of game objects
+        CSV.foreach('./spec/fixtures/fixture_games.csv', headers: true, header_converters: :symbol) do |row|
+            @game_objects << Games.new(
+              row[:game_id],
+              row[:season],
+              row[:away_team_id],
+              row[:home_team_id],
+              row[:away_goals].to_i,
+              row[:home_goals].to_i,
+            )
+        end
+    end
+
+    describe '#initialize' do
+        it 'exists' do
+            #test several elements in the game objects array to ensure they're all instances of Games
+            expect(@game_objects[0]).to be_an_instance_of(Games)
+            expect(@game_objects[1]).to be_an_instance_of(Games)
+            expect(@game_objects[2]).to be_an_instance_of(Games)
+            expect(@game_objects[3]).to be_an_instance_of(Games)
```

Comment:
This is great! In the future, know that you can iterate tests as well!:

@game_objects.each do |object|
   expect(object).to be_an_instance_of(Games)
end

Reply:
Oh very cool. Thanks for the suggestion!